persistence/relational-jdbc/src/main/java/org/apache/polaris/persistence/relational/jdbc/DatabaseType.java

```diff
@@ -43,7 +45,21 @@ public static DatabaseType fromDisplayName(String displayName) {
     };
   }
 
-  public String getInitScriptResource() {
-    return String.format("%s/schema-v1.sql", this.getDisplayName());
+  public String getInitScriptResource(@Nonnull SchemaOptions schemaOptions) {
+    final String schemaFile;
+    if (schemaOptions.schemaFile() != null) {
+      schemaFile = schemaOptions.schemaFile();
+    } else {
+      String schemaSuffix;
+      switch (schemaOptions.schemaVersion()) {
+        case SchemaOptions.LATEST -> schemaSuffix = "schema-v1.sql";
```

Comment:
suggestion: make `schemaVersion()` optional and default to latest.

Reply:
In SchemaOptions it is:
```
  @Value.Default
  default String schemaVersion() {
    return LATEST;
  }
```

Reply:
alternatively: use actual current schema version in the `LATEST` constant.

Reply:
how about `Optional<String> schemaVersion()`?

Reply:
I just changed to null by default, does this look better?

Reply:
Null works too :)

Reply:
Do we really need the `LATEST` constant now?

Reply:
I was thinking it's nice to have so that users can be explicit about taking the LATEST schema -- the alternative is just that you get LATEST by _not_ setting the version, right?

Besides the ability to be explicit about wanting the LATEST, I think the constant is potentially useful in the contrived scenario where a persistence wants to default to a non-LATEST schema

Reply:
Fair enough... but in this case I'd prefer to go back to the non-null `schemaVersion()` with `LATEST` as default in the Immutable class... Sorry about back-and-forth :facepalm: